glommio/src/executor/placement/linux_sysfs.rs

```diff
@@ -0,0 +1,837 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
there's already `glommio/src/sys/sysfs.rs`.
We should keep those things in a single location, and `src/sys` is better for that.

Sorry I haven't noticed this before, I think this code was already here in a previous version

Reply:
No, problem -- I'll have a look and move.